### ISSUE TITLE
[Bug]: Download tab and Generate M4B feature missing?

### ISSUE THREAD

**Issue by Kudjo-Grand** [advplyr/audiobookshelf#478]
### Describe the issue

I have been using AudioBookShelf for a while now and love how stable and feature-rich it has become.

I added a book the other day that had two files that I was going to use the built-in M4B generator. But, now I can't find it anymore. Did it get moved? I used that feature quite a lot. It had a few bugs, but was totally usable. I wondered if it got moved to the "Experimental Feature" set (which I usually have on, but it gets reset to off sometimes on an upgrade), but couldn't find it with that enabled either. Then I noticed that it looks like the enitre "Download" tab is missing from the Edit window. (See picture)

![135042517-597cb530-7c62-4259-b0db-935585e8f343](https://user-images.githubusercontent.com/25695143/164541263-1ee060d2-6628-419f-b224-3a0e3a674a4f.png)

Help me Obi-Wan Ke-, I mean...

If this was removed on purpose, would you consider adding it back (maybe as "experimental") so that I can keep using it? If it wasn't removed on purpose, then: TA-DA! I found a bug.

Thank you again for how awesome you have made ABS!

### Steps to reproduce the issue

1. Click on the Download icon that sits next to the "edit" pencil. 
2. Instead of opening the "download" tab on the Edit window, it takes you to the "detail" tab on the edit windows
3. Edit Window no longer has the "Download" tab on the far right anymore.

### Audiobookshelf version

2.0.0

### How are you running audiobookshelf?

Docker

**Comment by advplyr** [comment]
I think you were one of the first opening gh issues and many of your suggestions have been implemented. I appreciate your contributions over the last ~8 months.

I did remove that tab with the m4b merge tool because I wasn't happy with how it came out and it had some issues I was never able to track down. I can put it back in though and just make it experimental, I wasn't sure if it was being actively used so that is good to know.
